README.md

```diff
@@ -8,12 +8,7 @@ Here is the necessary section of `package.json`:
 ```javascript
 "devDependencies": {
     "mocha": "^5.1.1",
-    "eslint": "^4.0.0",
-    "eslint-config-standard": "^11.0.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.8.0",
-    "eslint-plugin-standard": "^3.1.0"
+    "eslint": "^4.0.0"
```

Comment:
I started thinking about this recommendation. They update major versions pretty frequently. The current one is 7.x. Do we try to keep up in here or change this to `latest`?

Reply:
I'll go ahead and make that `latest` recommendation for the sake of the PR discussion.

Reply:
Nevermind! We put a `peerDependencies` rule for `7.x` already.

Reply:
Couldn't that introduce a bc break to a version of eslint we're not compatible with?

Reply:
How so? I would think it would simply not know about a rule we're setting here. Given that this package updates major versions more frequently than most other Apostrophe projects, I would think the risk is minimal, but I'm probably not thinking of all the scenarios.

Reply:
Yeah. This is a meta tool, it doesn't have to work for apostrophe to work (or even for apostrophe to be secure), so I'm open to being more flexible about that.